### ISSUE TITLE
States Wishlist

### ISSUE THREAD

**Issue by SpicyAlfredo** [deliciousmods/1956_reboot#20]
Since we're starting from the ground up we have the freedom to add new states if we please.

- Though our focus is still currently on tech. We should start thinking of what states we want to add and get that debate flowing. And if we should even add states at all.

- **Let's try not to go hog wild with states.** Let's try to keep our additions down to a minimal. In the case if PDX ever messes with states again we'll have to update ours, and the PDX focus trees that reference them. 

- Also let's try to stay out of Europe if possible. With the exception of dividing up Wilnos 

- **Please give a good gameplay/Historical reason for adding a state.**

**Comment by SpicyAlfredo** [comment]
My wish-list: 

-USA 

West Virginia (Most likely will be added to game in "Man the Guns" but maintaining our own has benefits) Reason: Let's us have C.S.A and Union borders. Civil war anyone?

**I'm against dividing up New England into it's individual states. And adding Delaware. Cause there's not a good enough reason imo to do so.**

Texas could be divide in two. It's a more accurate reflection of Mexican claims but this is not a necessity 

-Africa 

Central Africa needs to be divide up into smaller pieces. Based on decolonization borders as much as possible. 

-China 

Some states need to be divide up to better reflect Tibet's claims in the region better. 

Some Northern states can be divide up to better reflect who had control of what. But pre-war china's politics are a mess and I'm no expert. So we should only divide these states up if we should have plans to add more warlords etc. Which I'm personally against. 

-Mongolia

Should probably be divide into two or three.  

-South America 

The amazon should be divide in relation of Peru's, Columbia claims into the region. Also I don't like a mega state like that. Though we should try to split it into two only.  

-Denmark 

Greenland should be divide into two. The usable coast line and it's ice sheet interior. (Impassable maybe?)

**Comment by Greatexperiment** [comment]
I agree, we should be very careful about this. There needs to be a very solid gameplay reason to add a state.

The gold standard for adding a state should be - there's something we really want to do that we can't do right now without altering the map. A great example of that would be the division of Berlin in the Cold War. It makes a lot of sense to make a new area to accurately show that and deal with the practical effects. I don't know that this particular example works well in our case (dividing it would be weird for WWII play) - but hopefully it illustrates the level of justification needed.

Dividing for the sake of dividing or just generally breaking up overly large provinces isn't a good enough reason. Regardless, this isn't something we're going to do for a very long time. There's lots of time to debate it. 

I admit that I am somewhat biased in favor of a more elegant map, rather than tons of provinces. It's one of the reasons I didn't get into HOI III - their map was basically porn for the hardcore segment, but a completely disaster of design, and definitely drove the message home - more is not always better.

**Comment by SpicyAlfredo** [comment]
I'm my nudge experiments I divide up French Africa into it's nation states. Since I think PDX wanted to that judging by the provinces but never did.

**Comment by Greatexperiment** [comment]
Africa is probably too simple to accommodate our plans to have more nations there. I'd say it makes sense to give them enough provinces to be functional and competitive.

**Comment by SpicyAlfredo** [comment]
![capturedsd](https://user-images.githubusercontent.com/38141958/41821626-ba754432-77b1-11e8-8102-c517b3b87938.JPG)
Here's what I've done with French Africa the other day.

**Comment by Lyylik51** [comment]
What I would like to see:
- Nordic states:
Historical states for the Nordic states, especially Sweden and Finland. The current states we have for Finland and Sweden are very random and way too large. Making them more historical would mean adding following states:
Finland: Uusimaa, Häme, Varsinais-Suomi, Mikkelin lääni
Sweden: Gälveborg, Västernorrland
Norway: Oppland-Hedmark, Finnmark, Kirkenes

**Comment by Greatexperiment** [comment]
Something to keep in mind is that PDX will probably add more states in Man the Guns - and the way the current system works means that it will royally screw with everyone else that adds them. In an ideal world we could just assign custom numbers to each one so the chance of conflict is minimal, but everything needs to be in order. :(

**Comment by eliasolsson** [comment]
After the most recent dev diary - I personally think Africa and China are the two regions to focus on for new states or provinces if at all. Some more impassable zones (especially in Siberia imo) could be nice.
